services/order.py

```diff
@@ -0,0 +1,29 @@
+from typing import List
+from django.db import transaction
+from db.models import Order, Ticket, User, MovieSession
+
+
+def create_order(tickets: List[dict],
+                 username: User.username,
+                 date=None):
+    with transaction.atomic():
+        user = User.objects.get(username=username)
+        order = Order.objects.create(user=user)
+
+        if date:
+            order.created_at = date
+            order.save()
+
+        for ticket in tickets:
+            Ticket.objects.create(
+                movie_session=MovieSession.objects.get(
+                    id=ticket["movie_session"]),
```

Comment:
You can also connect movie_session to ticket through id (movie_session_id=ticket["movie_session"]). It's simpler, but your variant is also good